index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CC Global Summit 49</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link href="media/favicon.ico" title="Icon" type="image/x-icon" rel="icon">
+</head>
+<body>
+    <header>
+        <section class="flexRow" id="top">
+            <img src="media/buttons/facebook.svg" alt="facebook handle">
+            <img src="media/buttons/twitter.svg" alt="twitter handle">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </section>
+        <section class="flexRow" id="nav">
+            <a href="index.html" id="logo">
+                <img src="media/logo.png" alt="cc global logo">
+            </a>
+            <a href="about.html"><p>About</p></a>
+            <a href="#"><p>Program</p></a>
+            <a href="#"><p>Join</p></a>
+            <a href="#"><p>Sponsor</p></a>
+            <a href="#"><p>News</p></a>
+            <button>CC Campaign</button>
+        </section>
+    </header>
+    <section id="welcome">
+        <img src="media/buttons/menu.svg" alt="menu" id="mobileMenuButton" onclick="mobileMenu()">
+        <div class="head">
+            <p>"Hello! Cooking world"</p>
+            <h1>COOKING CHANNEL GLOBAL SUMMIT 2049</h1>
+        </div>
+        <article>
+            <p>
+                Join us in this celebration of culinary delights!
+                (Mostly protein)
+                All the best chefs will be here. Yes, even that one.
+            </p>
+        </article>
+        <p id="date">2049.10.15(THU)~16(FRI)</p>
+        <p id="location">@ Wallace Convention Center, Wallace Art Center and more</p>
+    </section>
+    <section id="program">
+        <h2>Main Program</h2>
+        <div class="spacer"></div>
+        <div id="activities">
+        </div>
+        <button>Join CC Global Summit 2049</button>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="speakers">
```

Comment:
* I think The bottom section articles are organized in two columns of three articles each as it is in the original design 
![image](https://user-images.githubusercontent.com/52497246/128959933-bae8dfe6-a145-45b6-b540-284e145fc0c7.png)
*YOURS 
![image](https://user-images.githubusercontent.com/52497246/128959962-b3105f0e-977c-4ded-a337-219a2992117a.png)

Reply:
This result is the expected behaviour for a screen width smaller than 1247px; the elements would be really cramped if I put two columns in such a small screen size.

![ksnip_20210811-115645](https://user-images.githubusercontent.com/81324033/129071120-71a1bfd9-cd56-4021-bb0f-e2e89854998d.png)
 If you view the project on a screen wider than 1246px, you'll notice the elements sort themselves in columns as expected.

Reply:
Hello @AlexHamn, I noticed from 1231px, and below it shows one column, so it is common for a CR using a smaller screen size laptop to experience such, I think it would be nice to keep it 2 columns only for screens 768px and above then one column for 768px and below since these were the recommended sizes for distinguishing desktop and mobile screen sizes.
Take a look at this [design](https://defconuyo.netlify.app/), I think it's quite achievable.

Reply:
All right, thank you for the explanation 🙏🏼  
I have improved the code to adjust the articles in this section to be in two columns from 768px width onwards.

![ksnip_20210812-114448](https://user-images.githubusercontent.com/81324033/129235721-f224b594-fb30-4231-9132-52fbc0debd03.png)